addons/burdometer/burden.lua

```diff
@@ -1,8 +1,9 @@
 local os = require("os")
-local set = require("set")
-local player = require("player")
-local packet = require("packet")
+
```

Comment:
Apparently this file had it before, so this is not on you, but please replace all the double quotes with single quotes, both here and in other places of the file.

Reply:
Fixed by 2ced293.